integrations/deploy.yaml

```diff
@@ -562,14 +558,40 @@
   keywords:
     - windows
   install_description: |
-    1. Install [Windows Exporter](https://github.com/prometheus-community/windows_exporter) on every Windows host you want to monitor.
-    2. Install Netdata agent on Linux, FreeBSD or Mac.
-    3. Configure Netdata to collect data remotely from your Windows hosts by adding one job per host to windows.conf file. See the [configuration section](https://learn.netdata.cloud/docs/data-collection/monitor-anything/System%20Metrics/Windows-machines#configuration) for details.
-    4. Enable [virtual nodes](https://learn.netdata.cloud/docs/data-collection/windows-systems#virtual-nodes) configuration so the windows nodes are displayed as separate nodes.
+    Netdata offers a convenient Windows installer for easy setup. This executable provides two distinct installation modes, outlined below.
+
+    ## Graphical User Interface (GUI)
+
+    1. Download the Netdata [Windows installer](https://github.com/netdata/netdata-nightlies/releases) from the latest nightly release.
+    2. Run the `.exe` file and proceed with the installation process.
+    3. At a minimum, you will need your Netdata Cloud Space's claim token to connect your Agent to your Space.
+
+    ## Silent Mode (Command line)
+
+    If you prefer to install Netdata through the command line, you can do so by running the following command on Windows Powershell with administrator rights:
+
   methods:
-    - *ks_wget
-    - *ks_curl
-  additional_info: ""
+    - method: Silent Mode (Command line)
+      commands:
+        - channel: stable
+          command: >
+            $ProgressPreference = 'SilentlyContinue'; Invoke-WebRequest https://github.com/netdata/netdata-nightlies/releases/download/v1.99.0-267-nightly/netdata-installer-x64.exe -OutFile "netdata-installer-x64.exe"; .\netdata-installer-x64.exe /S /A {% if $showClaimingOptions %}/TOKEN={% claim_token %} /ROOMS={% $claim_rooms %}{% /if %}
```

Comment:
- This is supposed to be stable
- I don't like the hardcoded installer (`https://github.com/netdata/netdata-nightlies/releases/download/v1.99.0-267-nightly/netdata-installer-x64.exe`)

Reply:
Hey, talked with @netdata/product and we will change it once it is released as a stable release

Reply:
The plan is to not change anything between stable and nightly toggle atm, just point to the location where the user can get the exe

Reply:
@ilyam8 I de-hardcoded the version number.

In the future there will not be a nightly version, but only a stable one. So this issue persists, I don't think this will work if I leave stable empty.

Reply:
Can we not add stable if there is no stable?

Reply:
https://github.com/netdata/netdata/pull/18733#issuecomment-2401745828 check below

Reply:
@Ancairon : Right now we can add a Beta badge on the deploy integration and mention that it is only available on the nightly..

Reply:
> add a Beta badge
> only available on the nightly

And paid plans only too

Reply:
you mean something like 

> **Note**
>
> The Windows installer is still in beta, and thus available only in the nightly release channel

Reply:
@Ancairon it is not the installer in beta but the Agent, e.g.

> Netdata Windows Agent is still in beta and is only available on paid plans in our nightly release channel.